cmake/winml.cmake

```diff
@@ -616,7 +616,5 @@ endif()
 # When cuda is enabled in the pipeline, it sets CMAKE_SHARED_LINKER_FLAGS which affects all targets including winml_dll.
 # However, there are no cuda imports in winml_dll, and the linker throws the 4199 warning.
 # This is needed to allow winml_dll build with cuda enabled.
-set_target_properties(winml_dll
-  PROPERTIES
-  LINK_FLAGS
-  "/ignore:4199")
\ No newline at end of file
+#set_property(TARGET winml_dll APPEND_STRING PROPERTY LINK_FLAGS " /ignore:4199")
+target_link_options(winml_dll PRIVATE " /ignore:4199")
```

Comment:
This is adding `" /ignore:4199"` to the command line for me, no quotes is the correct way to do.

Reply:
"When cuda is enabled in the pipeline, it sets CMAKE_SHARED_LINKER_FLAGS which affects all targets including winml_dll." 

Then we can fix it, right? Or probably I already fix it?